launch/global_mapper_nodelet.launch

```diff
@@ -15,11 +15,12 @@
 
       <!-- remaps -->
       <remap from="cloud_topic" to="/flight/xtion_depth/points" />
-      <remap from="~map_topic" to="~global_map" />
+      <remap from="~pixel_map_topic" to="~pixel_map" />
+      <remap from="~voxel_map_topic" to="~voxel_map" />
```

Comment:
We have kind of settled on a best practice of including remaps for all publishers and subscribers.  This makes it very clear when something is not being remapped.